sql/src/main/java/io/crate/exceptions/ConversionException.java

```diff
@@ -28,32 +28,32 @@
 
 import java.util.Locale;
 
-public class ConversionException extends RuntimeException {
+public class ConversionException extends IllegalArgumentException {
 
     private final static String ERROR_MESSAGE = "cannot cast %s to type %s";
 
-    private String message;
+    public ConversionException(Symbol symbol, DataType targetType) {
```

Comment:
just a short cut to avoid the instanceof check if the caller knows what kind of object he has in hand